app/members/tracy-chen.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.njk' %}
+
+{% block title %}Tracy Chen | Bitcoin Cash Node{% endblock %}
+{% block description %}{% endblock %}
+
+{% block body %}
+    <section class="bg-dark">
+      <div class="container py-5">
+        <div class="row py-5 text-center text-md-left justify-content-center">
+          <div class="col-md-4 text-md-right">
+            <img class="rounded" src="/static/img/team/tracy_chen.jpg" alt="Tracy" width="200">
+          </div>
+          <div class="col-md-6 mt-5">
+            <h1 class="text-white font-weight-bold">Tracy Chen</h1>
+            <h4 class="text-white">BCHN Representative</h4>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section class="bg-light">
+      <div class="container py-5">
+        <div class="row my-5 justify-content-center">
+          <div class="col-md-9 text-lg">
+            <p>Tracy is the representative of BCHN, whose main role is
+            to establish contact channels and maintain communication on
+            behalf of BCHN team, BCH users and all ecological parties
+            (such as the exchange, mining pool, and wallet) to ensure
+            BCHN’s decisions to be fully approved and verified. She also
```

Comment:
Tracy is BCHN's public representative.
Her main role is to establish and maintain communication channels between the BCHN team and BCH users, stakeholders and the larger ecosystem such as exchanges, mining pools and wallets.
Tracy's work ensures that BCHN's decisions are communicated and vetted well with stakeholders.
She also...

Reply:
Thanks, updated